surveys/sphinx-survey-2025.yml

```diff
@@ -218,51 +204,6 @@ questions:
   - "XML/DITA"
   - "I don’t know"
 
-# Motivation: Understand the constellation of tools for documentation this responder works with.
-- title: "Which Python docstring style do you use?"
-  description: >-
-    Select N/A if you don’t use Python or Python docstrings.
-
-    Google-style docstrings look like:
-    ```rst
-    Arguments:
-        my_param (type): description
-    ```
-
-    Numpydoc-style docstrings look like:
-    ```rst
-    Arguments
-    ---------
-    my_param : type
-        description
-    ```
-
-    Sphinx-style docstrings look like:
-    ```rst
-    :param [my_param]: description
-    ```
-  multiple: true
-  other: true
-  choices:
-  - "Google format"
-  - "Numpydoc format"
-  - "Sphinx format"
-  - "Markdown"
-  - "I don’t know"
-  - "I do not write docstrings"
```

Comment:
I would still be interested in the result because it could tell priorities on the development of the napoleon extension. I agree though that the description is too lengthy.

Let's trim to this:

```
- title: "Which Python docstring style do you use?"
  multiple: true
  other: true
  choices:
  - "Google format"
  - "Numpydoc format"
  - "I don’t know"
  - "I do not write docstrings"
```

There may be more people in the "I don't know category". But that's ok. I also removed "Sphinx format" and "Markdown" because they are IMHO less and we have "other" anyway. @AA-Turner what do you think?

Reply:
I would be happy to keep these too.
Perhaps instead of the explicit syntax we can add a hyperlink to the examples in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html ?

Reply:
Both with or without the link would be ok for me.